### PR TITLE
133 add coin icons to assets

### PR DESCRIPTION
**Pull request by Vadim121197**

### CODE REVIEW

**Review by grod220 — COMMENTED** [review]
Instead of two components, can we just have one `<Identicon>` and it accepts a prop for either `gradient | solid`?

The areas it's called `IdenticonColor` we should call `identiconSolid`.

File: packages/ui/components/ui/identicon/generate.ts [hunk 1]
```diff
@@ -19,3 +19,25 @@ export const generateGradient = (str: string) => {
     toColor,
   };
 };
+
+export const generateColor = (str: string) => {
```
Comment (grod220) [inline packages/ui/components/ui/identicon/generate.ts:23]: Change this to `generateSolidColor`

**Review by grod220 — APPROVED** [review]

**Comment by Vadim121197** [comment]
Знімок екрана 2023-11-27 о 17 32 23

@grod220

**Comment by Vadim121197** [comment]
i can not merge

**Comment by grod220** [comment]
Screenshot 2023-11-27 at 3 44 21 PM

Just noticed that `gm` and `gn` coins have the same color. Given they have different names, this should not have happened right?

**Comment by Vadim121197** [comment]
Знімок екрана 2023-11-28 о 13 37 02

@grod220

**Comment by grod220** [comment]
Ah ok, bad luck then I suppose

**Pull request merged**